.github/workflows/integration-tests.yml

```diff
@@ -1,16 +1,12 @@
 name: Integration Tests
 
 on:
-  workflow_dispatch:
+  workflow_call:
     inputs:
       environment:
-        type: choice
+        type: string
         description: 'environment to run test in'
         required: true
-        options:
-          - staging
-          - alpha
-          - dev
         default: 'dev'
```

Comment:
Why are you switching to open text here?

Reply:
Choice seemed to not be an option for workflow_call - maybe I was misunderstanding the API docs though and there's another way to specify it?

Reply:
https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype